src/BaseServiceProvider.php

```diff
@@ -111,6 +113,11 @@ public function registerAdminMiddleware(Router $router)
         }
     }
 
+    public function registerCustomAuthGuard()
+    {
+        View::composer('backpack::*', \Backpack\Base\app\Http\ViewComposers\AuthComposer::class);
```

Comment:
Having both is probably the better way, given that Laravel does that too. We might be able to vavoid the arguments on slack/gitter bit though with a bit of luck (or just agreeing with me 👿).